perllib/FixMyStreet/App/Controller/Admin/Triage.pm

```diff
@@ -0,0 +1,108 @@
+package FixMyStreet::App::Controller::Admin::Triage;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'FixMyStreet::App::Controller::Reports'; }
```

Comment:
I think I'd prefer the lines in the function to call `/reports/...` than do this, because I'm not sure if this makes other pages available here that you don't want to?